hlx_statics/scripts.js

```diff
@@ -17,14 +17,14 @@
 //   }
 // };
 
-window.adobeid = {"client_id": "helix_adobeio","scope": "AdobeID,openid,read_organizations,additional_info.projectedProductContext,additional_info.roles,gnav,read_pc.dma_bullseye,creative_sdk", "locale": "en_US", "environment": "stg1", "useLocalStorage": false, "logsEnabled": true, redirect_uri: window.location.href};
+window.adobeid = {"client_id": "helix_adobeio","scope": "AdobeID,openid,unified_dev_portal,read_organizations,additional_info.projectedProductContext,additional_info.roles,gnav,read_pc.dma_bullseye,creative_sdk", "locale": "en_US", "environment": "stg1", "useLocalStorage": false, "logsEnabled": true};
```

Comment:
@timkim I didn't touch the following changes in scripts.js. Please check if these were your edit, or if I accidentally reverted them 😬 The other spacing updates are automatically fixed by the editor I use.

Reply:
Ah it's because I made some changes. Don't really need your edits in `scripts.js` so I'll just ignore those.